imsv-docs-docusaurus/openapi/endpoints/cards/models/card.yaml

```diff
@@ -20,45 +46,38 @@ properties:
       - cancelled
       - created
       - shipped
-  cardType:
+  fundingSourceIds:
+    type: array
+    description: List of Funding Source IDs linked to the card
+  expiry:
     type: string
-    description: The type of the card
-    enum:
-      - virtual
+    description: Expiry date of the card of physical cards
```

Comment:
@braden-holgate this one reads a little awkwardly, did you mean to say 'for physical cards'?